src/jit/lsra.h

```diff
@@ -1840,8 +1840,7 @@ class RefPosition
     // A RefPosition refers to either an Interval or a RegRecord. 'referent' points to one
     // of these types. If it refers to a RegRecord, then 'isPhysRegRef' is true. If it
     // refers to an Interval, then 'isPhysRegRef' is false.
-    //
-    // Q: can 'referent' be NULL?
+    // referent can never be null.
```

Comment:
This is unrelated but I happened upon this, and by construction `referent` is never null.